distributions/standalone/assembly-whitelist.txt

```diff
@@ -291,7 +291,7 @@
 /mule-standalone-${productVersion}/lib/opt/jasper-jdt-6.0.29.jar
 /mule-standalone-${productVersion}/lib/opt/javassist-3.7.ga.jar
 /mule-standalone-${productVersion}/lib/opt/javax.annotation-api-1.2.jar
-/mule-standalone-${productVersion}/lib/opt/javax.inject-2.3.0-b05.jar
+/mule-standalone-${productVersion}/lib/opt/javax.inject-1.jar
```

Comment:
it's not old... the one above is an OSGi repackaged version of the same library that is tied to the HK2 implementation that Jersey 2 uses. What I'm doing here is not depending on the Jersey (or HK2 for that matter) version. Instead, I'm using the official (and only) version of javax.inject